app/views/trainees/lead_schools/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1 class="govuk-heading-l">Lead school</h1>
+
+<%= register_form_with model: @trainee_lead_school_form, url: trainee_lead_schools_path(@trainee), local: true, html: { data: { module: "app-schools-autocomplete" } } do |f| %>
```

Comment:
This whole form (and potentially the heading above need to be restricted to a 2/3rds column as per the prototype.

Reply:
Changed to 2/3rds